backend/app/services/search.py

```diff
@@ -277,7 +287,10 @@ async def buildFilterData(self, filter, total):
 
             for key, value in refiner.items():
                 field = key
-                values = [bucket["key"] for bucket in value["buckets"]]
+                all_values = [bucket["key"] for bucket in value["buckets"]]
+                # to deduplicate values based on case-insensitive match
+                values = self.get_unique_values(all_values)
+                print(values)
```

Comment:
Don't leave `print` statements like this in the code ... they're not useful to anyone later without context. If something is *likely* to be generally useful for debugging later, you might do something like `print(f"buildFilterData refiner {values}")`, but even that's a bit opaque once you're no longer regularly staring at this code.

Somewhere along the line in my ILAB stuff I've added a Python `Logger`, and we should consider adding log debug/info/warn/error messages as appropriate over time, which will show module and function rather than just a value. For now, we should try to avoid just dumping arbitrary output on the server console that isn't really going to be much help to anyone.

Reply:
Got it! I’ll switch to using `Logger` from now on for better logging.